ldap/servers/plugins/replication/repl-session-plugin.h

```diff
@@ -43,8 +43,8 @@ typedef void *(*repl_session_plugin_agmt_init_cb)(const Slapi_DN *repl_subtree);
  * follows:
  *
  *   pre - send data to replica
- *   recv - receive data from master
- *   reply - send data to master
+ *   recv - receive data from supplier
+ *   reply - send data to supplier
```

Comment:
I agree with @droideck remark, here is should be 'receive data from sender' and 'send data to receiver'